test/contrib/k8s_job_test.py

```diff
@@ -62,22 +76,38 @@ class FailJob(KubernetesJobTask):
         }]
     }
 
+    def signal_complete(self):
+        with self.output().open('w') as output:
+            output.write('')
+
+    def output(self):
```

Comment:
Please remove this form this test, and test in a separate test case (SignalCompleteJob).